### DISCUSSION TITLE
Inheritance of managed classes in blueprints

### DISCUSSION THREAD

**Discussion by spo9kor** [nxrighthere/UnrealCLR#46] [Help] [answered]
I use c++ for unreal development , but I love c#. I am trying to inherit c# classes in blueprint, like I do with c++. please let me know if it is possible. by the way great work with this plugin. All matured developers want this plugin. it makes life easier.

Thanks for the great work.

**Comment by nxrighthere** [comment, marked as answer]
This feature is not supported (see my response [here](https://github.com/nxrighthere/UnrealCLR/issues/45)). The current features for blueprints interop are:
- Exposure of blueprint classes, functionality, and properties to managed code
- Exposure of managed functionality to blueprints (see https://github.com/nxrighthere/UnrealCLR/issues/33)

**Comment by spo9kor** [comment]
Thank you for the quick reply . I got it.
